.github/workflows/continuous-integration-workflow.yml

```diff
@@ -1,11 +1,14 @@
-name: Continuous integration
+name: Deploy
 on:
-  push:
-    branches:
-      - master
+  # Adds a 'Run Workflow' button to the GitHub Actions UI so the team can
+  # manually deploy the site.
+  workflow_dispatch:
+  
+  # Automatically build and deploy the site hourly,
+  # between the hours of 5am and 5pm
```

Comment:
Why only during these hours? web.dev contributors are spread across all kinds of timezones. Let's just deploy once per hour?

Reply:
Yeah I've gone back and forth on this. Originally I had it hourly Mon-Fri but was worried about something bad happening when the web.dev team was asleep (maybe a deploy that takes down the site or something).

But since the web.dev team is spread out between zurich-sf-sydney, we have quite a lot of overlap and could probably roll back easily enough.

What do you think about us not doing automatic deploys on Friday? I've gone back and forth on that one as well. I'd like to avoid us having to debug on the weekends if possible.

Reply:
My gut feeling would be to start with hourly deploys Mon-Fri, and then seeing what happens. It should be an _extremely_ rare event that a successful build (needed to trigger a deployment) results in the site being down, so I wouldn't optimize for it unless this actually occurs. Just my $0.02, it's totally your call!

Re: my original suggestion, making the deploy hourly 24 times a day would be more inclusive, timezone-wise.